module/private_quarters/private_quarters.py

```diff
@@ -0,0 +1,289 @@
+import module.config.server as server
+from module.base.timer import Timer
+from module.handler.assets import POPUP_CANCEL
+from module.logger import logger
+from module.ocr.ocr import Digit, DigitCounter
+from module.private_quarters.assets import *
+from module.ui.page import page_private_quarters
+from module.ui.ui import UI
+
+OCR_DAILY_COUNT = DigitCounter(PRIVATE_QUARTERS_DAILY_COUNT, letter=(255, 247, 247), threshold=64)
+if server.server != 'jp':
+    OCR_SHOP_GOLD_COINS = Digit(PRIVATE_QUARTERS_SHOP_GOLD_COINS, letter=(239, 239, 239), name='OCR_SHOP_GOLD_COINS')
+else:
+    OCR_SHOP_GOLD_COINS = Digit(PRIVATE_QUARTERS_SHOP_GOLD_COINS, letter=(201, 201, 201), name='OCR_SHOP_GOLD_COINS')
+
+class PrivateQuarters(UI):
+    # Key: str, target ship name
+    # Value: Button, button instance
+    available_targets = {
+        'anchorage': PRIVATE_QUARTERS_SHIP_ANCHORAGE,
+        'noshiro': PRIVATE_QUARTERS_SHIP_NOSHIRO,
+        'sirius': PRIVATE_QUARTERS_SHIP_SIRIUS,
+    }
+
+    def _pq_target_appear(self):
+        """
+        Callable wrapper to validate target's appearance
+        offset=(100, 100) detectable for anchorage, noshiro, and sirus
+        When more ships added may need to adjust or capture specific bubble position per
+        ship, can use the available_targets to store similarly into tuples instead
+        """
+        settle_timer = Timer(1.5, count=3).start()
+        settle_timer.wait()
+        self.device.screenshot()
+        return self.appear(PRIVATE_QUARTERS_ROOM_TARGET_CHECK, offset=(100, 100))
+
+    def _pq_goto_room_check(self):
+        """
+        Callable wrapper for whether is loading or blocked by download asset popup
+        """
+        return self.appear(PRIVATE_QUARTERS_LOADING_CHECK, offset=(20,20)) or self.appear(POPUP_CANCEL, offset=(20, 20))
+
+    def _pq_goto_room_enter(self, target_ship):
+        """
+        Execute enter room routine
+
+        Args:
+            target_ship (str):
+
+        Returns:
+            bool
+        """
+        # Initiate goto into target's room
+        # Ensure either loading or popup
+        # prompt appears after click
+        target_title = target_ship.title()
+        target_btn = self.available_targets[target_ship]
+        self.ui_click(
+            click_button=target_btn,
+            check_button=self._pq_goto_room_check,
+            appear_button=page_private_quarters.check_button,
+            offset=(20, 20),
+            skip_first_screenshot=False)
```

Comment:
always use skip_first_screenshot=True

Reply:
this might be reasonable, I'll test it if I recall the image it has on hand at the time should be sufficient for this operation of ui_click

Reply:
done in 3eb37033a17bb9af73fb16d984e61ca089e2f6a3